### ISSUE TITLE
A possible typo in utils.py

### ISSUE THREAD

**Issue by huizhang2017** [ZhangLingMing1/TSGCNet#5]
Hi, ZhangLingMing1,
      I am curious that why you didn't add "model.eval()" in the file 'utils.py' ? According to my understanding, during the validation stage or test stage, we usually use "model.eval()" first, then use 'torch.no_grad()', which will fix the BN and dropout, and do not calculate the gradient.  Is this a typo?

**Comment by 2667402485chen** [comment]
> Hi, ZhangLingMing1, I am curious that why you didn't add "model.eval()" in the file 'utils.py' ? According to my understanding, during the validation stage or test stage, we usually use "model.eval()" first, then use 'torch.no_grad()', which will fix the BN and dropout, and do not calculate the gradient. Is this a typo?

Hi, can you please put data into this network, I am also looking at this paper recently. There are some questions I want to ask you, is it convenient to give me your contact information, so that we can communicate and study together, or you can add my qq, my qq is 2667402485, looking forward to your reply.
